docs/get_started/getting_started_tensorflow.md

```diff
@@ -0,0 +1,81 @@
+# Getting Started with IREE TensorFlow Integrations
```

Comment:
Eh, really not a fan of there being even _more_ "getting started" docs... we should really differentiate more between developer docs and user docs, and anything involving building from source is pretty developer-oriented :P

What concerns me in particular about this document (where it is located and how it is titled) is that users coming from TensorFlow Python should absolutely _not_ be following this guide - they should be using the pip packages.

Reply:
The contents of the doc are great though :) ... just concerned with the direction the "getting started" docs are taking.

Reply:
I welcome anyone writing docs on how to do this :stuck_out_tongue: This is the only way I know how to do it. Note that I do have a section for getting the binaries from the pip packages. All our getting started docs are about building from source. Until quite recently that was the only thing to get started on. But in particular @ewconnell was trying to actually use this stuff and the current documentation is super out of date, so I thought I would try to at least not make that *worse* with this change.

Reply:
(and because tone is hard to convey, I don't disagree with you here). I think the addition of this doc makes things at least better documented, so we should keep it. The current docs are just...wrong AFAICT

Reply:
I think we need to re-organize a lot of these docs into a longer "developer guide" instead of discrete getting started docs. We should reserve the "getting started" moniker for actual users, not us poor saps who build this stuff.

Fine to reorganize as a followup.